helm/estafette-vulnerability-scanner/templates/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
Please add `{{- if .Values.serviceMonitor.enabled }}` here as well, since the application doesn't need a service for any other purposes.